lib/taskana-core/src/test/java/pro/taskana/database/TestDataGenerator.java

```diff
@@ -43,6 +44,9 @@ public void generateTestData(DataSource dataSource) throws SQLException {
         ScriptRunner runner = null;
         try {
             Connection connection = dataSource.getConnection();
+            if (connection.getMetaData().getDatabaseProductName().contains("DB2")) {

```

Comment:
Isn't it smarter to create a static method TaskanaEngineImpl.isDb2(String dbProductName) so that the identification of a db2 database is only located in one space?